components/crud-web-apps/common/frontend/kubeflow-common-lib/projects/kubeflow/src/lib/status-icon/status-icon.component.scss

```diff
@@ -1,15 +1,25 @@
 .small-padding-right {
   padding-right: 8px;
```

Comment:
@elenzio9 same thing I mention [here](https://github.com/kubeflow/kubeflow/pull/6952#discussion_r1132106601) also applies in this padding

Reply:
Done! Just pushed the changes!